packages/manager/apps/user-account/src/app.module.js

```diff
@@ -0,0 +1,50 @@
+import 'script-loader!jsurl/lib/jsurl.js'; // eslint-disable-line
+import 'script-loader!qrcode.js/lib/qrcode.js'; // eslint-disable-line
+import 'script-loader!ipaddr.js/ipaddr.min.js'; // eslint-disable-line
+
+import angular from 'angular';
+import 'angular-ui-bootstrap';
+import 'angular-qr';
```

Comment:
i don't think you are up to date with develop because i did a specific fix to lazy load angular-qr :
https://github.com/ovh/manager/pull/3996

Reply:
I did not rebase with develop, ill check for new changes, thanks